tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
```

Comment:
```
{ '@Bar': false, } : -@Bar => { 'AA' => true, NOOP' => false, } // as direct revert
```

this is revertion by simple rule
> if your configuration only includes a disabled ruleset with no further rules, the resolved result should be an empty list. In your exemple, this enables AA rule.

it would make exception of how to understand sets and resolve them.
if set contains only rule disabling and you don't want to disable or enable them - don't use the set